### DISCUSSION TITLE
Canvasblocker in FF w/ private browsing

### DISCUSSION THREAD

**Discussion by ajay-gh** [kkapsner/CanvasBlocker#540] [Q&A]
For obvious reasons FF does not offer container tabs when in private browsing mode. But CanvasBlocker is running in the background, and the navigator API is necessarily using a container to determine which data to present. Is this always the "first" container ? By first I mean the no-name container shown on top of the list in navigator settings, or does it use some other defaults ? Thanks

**Comment by kkapsner** [comment]
Yes, it should use the default (no-name) container. You can check on https://canvasblocker.kkapsner.de/test/navigatorTest.php to see what values are used (the red warnings are because of #473) .
